`kraft rm` without argument panics

### Describe the bug

```console
$ sudo "$(which kraft)" rm
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
kraftkit.sh/cmd/kraft/rm.(*Rm).Run(0xc0001236b0, 0x0?, {0x0, 0x0, 0x0?})
        /home/acotten/git/unikraft/kraftkit/cmd/kraft/rm/rm.go:107 +0x168f
kraftkit.sh/cmdfactory.bind.func1(0xc0006ae300?, {0x0, 0x0, 0x0})
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:687 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0xc0006ae300?, {0x0, 0x0, 0x0})
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:687 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0xc0006ae300?, {0x0, 0x0, 0x0})
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:687 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0x0?, {0x0, 0x0, 0x0})
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:687 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0xc00081fd98?, {0x0, 0x0, 0x0})
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:687 +0x19d
kraftkit.sh/cmdfactory.execute(0xc0006ae300, {0x3381e10, 0x0, 0x0})
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:229 +0x30b
kraftkit.sh/cmdfactory.executeC(0xc000004300)
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:289 +0x205
kraftkit.sh/cmdfactory.Main({0x2601048?, 0xc000437e60}, 0xc000004300)
        /home/acotten/git/unikraft/kraftkit/cmdfactory/builder.go:320 +0x73
main.main()
        /home/acotten/git/unikraft/kraftkit/cmd/kraft/kraft.go:143 +0x2b9
```

--- a/cmd/kraft/rm/rm.go
+++ b/cmd/kraft/rm/rm.go
@@ -60,6 +60,10 @@ func (opts *Rm) Pre(cmd *cobra.Command, _ []string) error {
 func (opts *Rm) Run(cmd *cobra.Command, args []string) error {
 	var err error
 
+	if len(args) == 0 && !opts.All {
+		return fmt.Errorf("no machine(s) specified")
+	}
+
 	ctx := cmd.Context()
 	platform := mplatform.PlatformUnknown
 	var controller machineapi.MachineService

